index.d.ts

```diff
@@ -471,6 +499,36 @@ declare namespace Moleculer {
 		}
 	}
 
+	class Transit {
```

Comment:
I've made the assumption that all the function starting with '_' are internal only so I didn't include them.